app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/Dialog.kt

```diff
@@ -36,26 +36,26 @@ import com.hedvig.android.design.system.hedvig.tokens.DialogTokens
 import hedvig.resources.R
 
 @Composable
-fun HedvigDialogError(
-  titleText: String,
-  descriptionText: String,
-  buttonText: String,
-  onButtonClick: () -> Unit,
-  onDismissRequest: () -> Unit,
+fun ErrorDialog(
```

Comment:
The one below is named `HedvigAlertDialog`, it might be a bit confusing to have them named differently perhaps?
What if we also named this `HedvigErrorDialog`?

Reply:
The thing is, those are named ErrorDialog in the old design. For the sake of easier transition, I'd say let's leave the naming as it was int he old one maybe, and then rename after the migration?